lib/elixir_internal_certificate_web/controllers/api/auth_error_handler.ex

```diff
@@ -0,0 +1,16 @@
+defmodule ElixirInternalCertificateWeb.Api.AuthErrorHandler do
+  import Plug.Conn
+  import Phoenix.Controller
+
+  alias ElixirInternalCertificateWeb.Api.ErrorView
+
+  def auth_error(conn, {type, _reason}, _opts) do
+    conn
+    |> put_status(:unauthorized)
+    |> put_view(ErrorView)
+    |> render("error.json", %{
+      code: "unauthorized",
+      detail: to_string(type)
```

Comment:
How about we just simply the message here 

```suggestion
      detail: "Login required"
```

Reply:
Updated in [1c3cf1a](https://github.com/nvminhtue/elixir_internal_certificate/pull/44/commits/1c3cf1ac992081b443e4ab1840f95cb2fbce3f0b)